CinnVIIStarkMenu@NikoKrause/po/CinnVIIStarkMenu.pot

```diff
@@ -603,7 +603,7 @@ msgstr ""
 
 #. 3.2->settings-schema.json->show-favorite-icons->description
 #. 2.6->settings-schema.json->show-favorite-icons->description
-msgid "Show favorite icons"
+msgid "Show icons on favorites"
```

Comment:
Your wording sounds better.
But I want this to be analog to 
Show application icons, etc